<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeaWindow</class>
 <widget class="QMainWindow" name="TeaWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1050</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1050</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1000</width>
      <height>650</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1000</width>
      <height>650</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>650</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;华文仿宋&quot;;</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>查看学生请假列表</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>801</width>
        <height>591</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;华文仿宋&quot;;</string>
      </property>
      <column>
       <property name="text">
        <string>编号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>姓名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>学号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>班级</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>请假原因</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>请假时间</string>
       </property>
      </column>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>10</y>
        <width>161</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string>操作</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>141</width>
         <height>121</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_agree">
          <property name="text">
           <string>同意</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_disagree">
          <property name="text">
           <string>不同意</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_quit">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>570</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>180</y>
        <width>161</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>检索</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>141</width>
         <height>81</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QComboBox" name="comboBox_type">
          <item>
           <property name="text">
            <string>姓名</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>班级</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>学号</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="lineEdit_type"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_shuaxin">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>330</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>刷新列表</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>批准列表</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>801</width>
        <height>591</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;华文仿宋&quot;;</string>
      </property>
      <column>
       <property name="text">
        <string>编号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>姓名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>学号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>班级</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>请假原因</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>请假时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>审批时间</string>
       </property>
      </column>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>10</y>
        <width>161</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>检索</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>141</width>
         <height>91</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QComboBox" name="comboBox_type2">
          <item>
           <property name="text">
            <string>姓名</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>班级</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>学号</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="lineEdit_type2"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_quit_2">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>570</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>拒绝列表</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>10</y>
        <width>161</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>检索</string>
      </property>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>141</width>
         <height>91</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QComboBox" name="comboBox_type3">
          <item>
           <property name="text">
            <string>姓名</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>班级</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>学号</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="lineEdit_type3"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_quit_3">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>570</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>801</width>
        <height>591</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;华文仿宋&quot;;</string>
      </property>
      <column>
       <property name="text">
        <string>编号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>姓名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>学号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>班级</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>请假原因</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>请假时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>拒绝理由</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>审批时间</string>
       </property>
      </column>
     </widget>
     <zorder>tableWidget_3</zorder>
     <zorder>groupBox_4</zorder>
     <zorder>pushButton_quit_3</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
